game-core/src/main/java/games/strategy/triplea/ui/PoliticsPanel.java

```diff
@@ -57,61 +55,81 @@
-                              + (overviewScroll.getPreferredSize().width > availWidth ? 20 : 0)))));
+                  new PoliticalStateOverview(getData(), getMap().getUiContext(), false) {
+
+                    private GridBagConstraints makeGridBagConstraints(
+                        final Insets insets, final int row) {
+                      return new GridBagConstraints(
+                          1,
+                          row,
+                          1,
+                          1,
+                          1.0,
+                          1.0,
+                          GridBagConstraints.WEST,
+                          GridBagConstraints.HORIZONTAL,
+                          insets,
+                          0,
+                          0);
+                    }
+
+                    @Override
+                    protected JPanel getRelationshipComponent(
+                        final PlayerId player1,
+                        final PlayerId player2,
+                        final RelationshipType relType,
+                        final Color relColor) {
+                      final JPanel panel = new JPanel();
+                      panel.setOpaque(false);
+                      panel.setLayout(new GridBagLayout());
 
-              final JScrollPane choiceScroll =
-                  new JScrollPane(politicalActionButtonPanel(politicalChoiceDialog));
-              choiceScroll.setBorder(BorderFactory.createEmptyBorder());
-              choiceScroll.setPreferredSize(
-                  new Dimension(
-                      (choiceScroll.getPreferredSize().width > availWidth
-                          ? availWidth
-                          : (choiceScroll.getPreferredSize().width
-                              + (choiceScroll.getPreferredSize().height > availHeightChoice
-                                  ? 20
-                                  : 0))),
-                      (choiceScroll.getPreferredSize().height > availHeightChoice
-                          ? availHeightChoice
-                          : choiceScroll.getPreferredSize().height
-                              + (choiceScroll.getPreferredSize().width > availWidth ? 20 : 0))));
+                      final Insets insets = new Insets(1, 1, 1, 1);
+                      int row = 0;
+                      final JPanel status = wrapInJPanel(new JLabel(relType.getName()), relColor);
+                      panel.add(status, makeGridBagConstraints(insets, row++));
+                      if (!player1.equals(getCurrentPlayer())) {
+                        return panel;
+                      }
 
-              final JSplitPane splitPane =
-                  new JSplitPane(JSplitPane.VERTICAL_SPLIT, overviewScroll, choiceScroll);
-              splitPane.setOneTouchExpandable(true);
-              splitPane.setDividerSize(8);
+                      for (final PoliticalActionAttachment paa : validPoliticalActions) {
```

Comment:
nit: abbreviations should probably be avoided: `paa` and favor spelling it out instead. I know it's existing, but given the relatively heavy modifications in this section, I think it probably makes sense to go ahead and fix it.

The cost to write this variable out fully is one-time, meanwhile it helps anyone reading the code (it's one less thing for them to keep track of in their mind when trying to scan/or study this section of code).